pants-plugins/src/python/internal_backend/sitegen/tasks/sitegen.py

```diff
@@ -354,7 +355,9 @@ def generate_generated(config, here):
 
 def render_html(dst, config, soups, precomputed, template):
   soup = soups[dst]
-  renderer = Renderer()
+  renderer = Renderer(escape=cgi.escape)  # TODO(python3port): cgi.escape is deprecated in favor html.escape
```

Comment:
This was a tough bug to find. Pystache by default uses `cgi.escape()` in Python2, but `html.escape()` in Python3. Because `future` adds a backport to `html` (which doesn't exist in Python2), Pystache successfully was importing the module and using `html.escape()`.

`html.escape()` escapes `'` unlike `cgi.escape()`, so one of our tests was failing. 

For now, I'm having us stay with the old semantics.